lib/util/pssh.js

```diff
@@ -31,8 +31,9 @@ goog.require('shaka.util.Uint8ArrayUtils');
 shaka.util.Pssh = class {
   /**
    * @param {!Uint8Array} psshBox
+   * @param {boolean=} extractMoov boolean to extract pssh from moov atom
```

Comment:
Perhaps `extractFromMoov` would be a better name, then?

Or, perhaps remove this argument, both here and in containsPssh.  It seems that the mp4 parser will still parse a pssh box at the root if you define both it and moov.  There may be no need for this.

Reply:
Done! I rename it to extractFromMoov